scionlab/templates/scionlab/partials/installation_type_pkg.html

```diff
@@ -1,5 +1,5 @@
 <p>
-To install SCION from packages, please follow the steps in the <a href="https://netsec-ethz.github.io/scion-tutorials/install/pkg/" target="_blank">tutorials</a>.
+To install SCION from packages, please follow the steps in the <a href="https://docs.scionlab.org/content/install/pkg.html" target="_blank">tutorials</a>.
```

Comment:
Speaking of this... Do we want to change the settings to not have the `.html` in the URLs? (I can do it, let me know whether I should.)

Reply:
Now you can (and should) remove `.html` :D